test/Plutip/UtxoDistribution.purs

```diff
@@ -75,15 +75,15 @@ import Type.Prelude (Proxy(Proxy))
 suite :: TestPlanM (Aff Unit) Unit
 suite = group "UtxoDistribution" do
   test
-    "runPlutipContract: stake key transfers with distribution: [[1000000000,1000000000]]"
+    "stake key transfers with distribution: [[1000000000,1000000000]]"
```

Comment:
Are we changing this in every test?

Reply:
We already have - the PR that changed this missed the additions from my earlier PR